### PR TITLE
Register user with expected values

### PR DESCRIPTION
**Pull request by gerigol**

### CODE REVIEW

File: src/frontend/src/register/Register.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import {Button, Form, Input} from 'antd';
+
+const onSubmitFailed = (errorInfo) => {
+    console.log('Failed:', errorInfo);
```
Comment (MateWagner) [inline src/frontend/src/register/Register.js]: Maybe a Notice on the page what happen why is the error

File: src/frontend/src/register/Register.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+};
+
+const onSubmit = (values) => {
```
Comment (MateWagner) [inline src/frontend/src/register/Register.js]: whats up with the async code ? and the returned response can trigger an error message and if the registration was successful then navigate else ware

File: src/frontend/src/register/Register.js (hunk 1 above)
Comment (gerigol) [reply src/frontend/src/register/Register.js]: done using the response message

File: src/frontend/src/register/Register.js (hunk 2 above)
Comment (gerigol) [reply src/frontend/src/register/Register.js]: done useing antD to show pop up on successness of post request

**Pull request merged**